client/disputes/evidence/dispute-evidence-page.js

```diff
@@ -0,0 +1,138 @@
+/**
```

Comment:
This code used to be in the `client/disputes/evidence/index.js` file, but I moved it here to make that file a bit cleaner. I made some minor tweaks to the code to use data from the wp.data state tree, but it's mostly the same.